### PR TITLE
feat(CL): CreatePosition and WithdrawPosition messages

### PR DESCRIPTION
**Pull request by p0mvn**
Progress towards: #3134

## What is the purpose of the change

Adding LPing messages.

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? no
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? no
  - How is the feature or change documented? not applicable

### CODE REVIEW

**Review by czarcas7ic — APPROVED** [review]
👍

File: x/concentrated-liquidity/lp.go [hunk 1]
```diff
@@ -8,7 +8,10 @@ import (
 	types "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity/types"
 )
 
-func (k Keeper) Mint(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, liquidityIn sdk.Dec, lowerTick, upperTick int64) (amtDenom0, amtDenom1 sdk.Int, err error) {
+func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, amount0Desired, amount1Desired, amount0Min, amount1Min sdk.Int, lowerTick, upperTick int64) (amtDenom0, amtDenom1 sdk.Int, err error) {
+	// TODO: calculate from amounts given
+	liquidityIn := sdk.MustNewDecFromStr("1517.882323")
```
Comment (mattverse) [inline x/concentrated-liquidity/lp.go:13]: why are we using hard coded values?

File: x/concentrated-liquidity/lp_test.go [hunk 2]
```diff
@@ -17,21 +17,23 @@ func (s *KeeperTestSuite) TestMint() {
 	currentTick := sdk.NewInt(85176)
 	lowerTick := int64(84222)
 	upperTick := int64(86129)
-	liquidity, err := sdk.NewDecFromStr("1517.882323")
```
Comment (mattverse) [inline x/concentrated-liquidity/lp_test.go:20]: why move this testing params from test to keeper layer?

File: x/concentrated-liquidity/types/events.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	TypeEvtWithdrawPosition = "withdraw_position"
+
+	AttributeValueCategory = ModuleName
+	AttributeKeyPoolId     = "pool_id"
+	AttributeAmount0       = "amount0"
+	AttributeAmount1       = "amount1"
+	AttributeLiquidity     = "liquidity"
+	AttributeLowerTick     = "lower_tick"
+	AttributeUpperTick     = "upper_tick"
+)
```
Comment (mattverse) [inline x/concentrated-liquidity/types/events.go:14]: can we add these later when we use it?

File: x/concentrated-liquidity/lp_test.go (hunk 2 above)
Comment (p0mvn) [reply x/concentrated-liquidity/lp_test.go:20]: API has changed. This is the final API that we are going to have

File: x/concentrated-liquidity/lp.go (hunk 1 above)
Comment (p0mvn) [reply x/concentrated-liquidity/lp.go:13]: Synched offline

Comment (p0mvn) [reply x/concentrated-liquidity/lp.go:13]: To make tests pass. This value has to be calculated from amounts given

File: x/concentrated-liquidity/types/events.go (hunk 3 above)
Comment (p0mvn) [reply x/concentrated-liquidity/types/events.go:14]: synched offline

**Pull request merged**